migrations/project_aliases.go

```diff
@@ -0,0 +1,101 @@
+package migrations
+
+import (
+	"github.com/mongodb/anser"
+	"github.com/mongodb/anser/db"
+	"github.com/mongodb/anser/model"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
+)
+
+const (
+	projectVarsCollection  = "project_vars"
```

Comment:
These are also in the model. Is there a reason these need to be here?

Reply:
We want to avoid dependencies between migrations and model. Suppose I write a migration today that depends on a struct's existing structure, and then I change that structure a year from now. That could break the migration in the future, or attempt to migrate an already migrated document, which we don't want.